content/sensu-go/5.15/api/federation.md

```diff
@@ -0,0 +1,326 @@
+    "replication_interval_seconds": 5
+  }
+}
+{{< /highlight >}}
+response codes  | <ul><li>**Success**: 201 (Created)</li><li>**Malformed**: 400 (Bad Request)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+
+### `/etcd-replicators/:etcd-replicator` (DELETE) {#etcd-replicatorsetcd-replicator-delete}
+
+The `/etcd-replicators/:etcd-replicator` endpoint provides HTTP DELETE access to delete the specified replicator from Sensu.
+
+### EXAMPLE {#etcd-replicatorsetcd-replicator-delete-example}
+
+The following example shows a request to delete the replicator `my_replicator`, resulting in a successful HTTP 204 No Content response.
+
+{{< highlight shell >}}
+curl -X DELETE \
+-H "Authorization: Bearer $SENSU_TOKEN" \
+http://127.0.0.1:8080/api/federation/v1/etcd-replicators/my_replicator
+
+HTTP/1.1 204 No Content
+{{< /highlight >}}
+
+#### API Specification {#etcd-replicatorsetcd-replicator-delete-specification}
+
+/etcd-replicators/:etcd-replicator (DELETE) | 
+--------------------------|------
+description               | Deletes the specified replicator from Sensu.
+example url               | http://hostname:8080/api/federation/v1/etcd-replicators/{etcd-replicator_name}
+response codes            | <ul><li>**Success**: 204 (No Content)</li><li>**Missing**: 404 (Not Found)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+
+
+## The `/cluster` endpoint
+
+### `/cluster` (GET)
+
+The `/cluster` endpoint provides HTTP GET access to a list of clusters.
+
+#### EXAMPLE {#cluster-get-example}
+
+The following example demonstrates a request to the `/cluster` endpoint, resulting in a list of clusters.
+
+{{< highlight shell >}}
+curl http://127.0.0.1:8080/api/federation/v1/cluster -H "Authorization: Bearer $SENSU_TOKEN"
+[
+  {
+    "api_version": "federation/v1",
+    "type": "cluster",
+    "metadata": {
+      "name": "my_cluster"
+    },
+    "spec": {
+      "api_urls": {
+        "url": "http://10.0.0.1:8080",
+        "url": "http://10.0.0.2:8080",
+        "url": "http://10.0.0.3:8080"
+      }
+    }
+  }
+]
+{{< /highlight >}}
+
+#### API Specification {#cluster-get-specification}
+
+/cluster (GET)  | 
+---------------|------
+description    | Returns a list of clusters.
+example url    | http://hostname:8080/api/federation/v1/cluster
+response type  | Array
+response codes | <ul><li>**Success**: 200 (OK)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+output         | {{< highlight shell >}}
+[
+  {
+    "api_version": "federation/v1",
+    "type": "cluster",
+    "metadata": {
+      "name": "my_cluster"
+    },
+    "spec": {
+      "api_urls": {
+        "url": "http://10.0.0.1:8080",
+        "url": "http://10.0.0.2:8080",
+        "url": "http://10.0.0.3:8080"
+      }
+    }
+  }
+]
+{{< /highlight >}}
+
+## The `/cluster/:cluster` endpoint {#the-clustercluster-endpoint}
+
+_**NOTE**: Only cluster admins have PUT and DELETE access to clusters._
+
+### `/cluster/:cluster` (PUT) {#clustercluster-put}
+
```

Comment:
It looks like we're missing a description and example for the PUT endpoint.

Reply:
Yeah -- I don't have an example for either of the PUT endpoints for federation (etcd-replicators or clusters)